build/Dockerfile

```diff
@@ -20,6 +20,10 @@ ENV OPERATOR=/usr/local/bin/kubernetes-nmstate \
 # install operator binary
 COPY build/_output/bin/kubernetes-nmstate ${OPERATOR}
 
+# Copy webhook ssl
+COPY build/_output/nmstate-webhook.pem /ssl/tls.crt
```

Comment:
Please make it so we load these from a secret. It does not need to be perfect, but we better build it with cert-manager in mind. Since you mount a secret as a file, it should remain the same for the app.

Reply:
https://github.com/morvencao/kube-mutating-webhook-tutorial/blob/master/deployment/webhook-create-signed-cert.sh 

this can help for that ?

Reply:
I will do this as next step since, I see the hook callback working now, thanks for the pointer  @oshoval

Reply:
Ok done, @phoracek can you take a look ?